.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: scala
+
+jdk:
+  - oraclejdk8
+
+scala:
+  - 2.10.5
```

Comment:
@helena Shouldn't we maintain this in line with crossScalaVersions in build.sbt? (I do not think it should make a difference now since it is only a minor difference.)
`crossScalaVersions := Seq("2.11.7", "2.10.6")`